deploy/FullDeploy.js

```diff
@@ -165,8 +193,10 @@ module.exports = async function (hre) {
         log: true,
     })
 
+    console.log(`Factory Deployed: ${factory.address}`)
+
     let maxLeverage = ethers.utils.parseEther("12.5")
-    let tokenDecimals = new ethers.BigNumber.from("8").toString()
+    let tokenDecimals = new ethers.BigNumber.from("18").toString()
```

Comment:
Is this correct? I think the quote token hasn't been modified from its default 8 decimals.

Reply:
@BenjaminPatch the quote token (`TestToken.sol`) doesn't take in a decimals option so it defaults to 18 I am pretty sure. We also pass in the totalSupply as a 18 decimal number

Reply:
Yep that makes sense. I was getting this mixed up with the price oracle.